apps/etterlatte-behandling/src/main/kotlin/behandling/manueltopphoer/ManueltOpphoerService.kt

```diff
@@ -57,22 +58,24 @@ class RealManueltOpphoerService(
         opphoerRequest: ManueltOpphoerRequest
     ): ManueltOpphoer? {
         return inTransaction {
-            val forrigeBehandling =
-                behandlinger.alleBehandingerISak(opphoerRequest.sak).`siste ikke-avbrutte behandling`()
+            val alleBehandlingerISak = behandlinger.alleBehandlingerISak(opphoerRequest.sak)
+            val forrigeBehandling = alleBehandlingerISak.`siste ikke-avbrutte behandling`()
 
             when (forrigeBehandling) {
                 is Foerstegangsbehandling -> ManueltOpphoer(
                     sak = forrigeBehandling.sak,
                     persongalleri = forrigeBehandling.persongalleri,
                     opphoerAarsaker = opphoerRequest.opphoerAarsaker,
-                    fritekstAarsak = opphoerRequest.fritekstAarsak
+                    fritekstAarsak = opphoerRequest.fritekstAarsak,
+                    virkningstidspunkt = forrigeBehandling.virkningstidspunkt
```

Comment:
Burde ikke virkningstidspunkt for manuelt opphør være fra og med første innvilgete førstegångsbehandling på saken? Ser for meg at forrige behandlingens virkningstidspunkt kun holder for f.eks 2 behandlinger på saken. Eller tenker jeg feil?

Reply:
Hvis den forrige behandlingen er en førstegangsbehandling, er det vel bare den vi skal opphøre? Det er i tilfellene der vi har revurderinger at logikken blir mer komplisert (se under). Jeg kan jo forsåvidt endre begge til å være `alleBehandlingerISak.tidligsteIverksatteVirkningstidspunkt()`, siden det ikke er feil for førstegangsbehandlinger heller